tools/osbuild-mpp

```diff
@@ -666,11 +693,19 @@ class ManifestFile:
         json.dump(self.root, file, indent=2, sort_keys=sort_keys)
         file.write("\n")
 
+    def _rewrite_node (self, node):
+        fakeroot = [node]
+        self._process_format(fakeroot)
+        if len(fakeroot) == 0:
+            return (None, True)
```

Comment:
```suggestion
            return None, True
```

You don't need `( )` I think. Here and in all the other place. Maybe in the future we should have a `namedtuple` so that is clearer what is going on here.